File: Programming_c++/Assignments/Lessons[24-29]/A8.cpp
#include "main.h"

int main()
{
    int num = 10;

    cout << --num << " | " << num++ << " | " << ++num << "\n"; // 9 | 9 | 11
    cout << num << " | " << ++num << " | " << num << "\n"; // 11 | 12 | 12
    cout << --num << " | " << --num << " | " << --num << "\n"; // 11 | 10 | 9
}

File: Front_end_dev/JAVASCRIPT/Assignments on Whole Bootcamp/A8.js
function findTheClosest(nums, goal) {
    let diffs = [];

    for (let num of nums) {
        diffs.push(goal - num);
    }

    return nums[diffs.indexOf(Math.min(...diffs))];
}

let nums = [10, 80, 85, 25, 30, 88, 15];
let goal = 100;

console.log(findTheClosest(nums, goal));

// Closest Number Is 88